app/views/invitations/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <!--Descriptive page name, messages and instructions-->
```

Comment:
I'm having a little trouble reproducing this one. The form wrappers invitation i.e `form_for(invitation)` so it should be submitting the `invitation` parameter. Could you possible list the steps to reproduce this?